ui/src/translations/fr.json

```diff
@@ -37,18 +37,23 @@
   "deployment": "Déploiement",
   "version": "MetalK8s Version",
   "node_deployment": "Déploiement du node",
```

Comment:
Not really related to this PR (but since you're updating the translation…), but:
- do we really want to use `grappe` instead of `cluster`?
- on the other hand, we should clearly use `nœud` instead of `node`
- in the English strings, we have `At least on role has to be selected!` => `s/on role/one role/`.
-

Reply:
I'll create another ticket to review all translations https://github.com/scality/metalk8s/issues/1192